cmd/system-probe/subcommands/run/command.go

```diff
@@ -298,7 +299,8 @@ func runSystemProbe(ctxChan <-chan context.Context, errChan chan error) error {
 			RemoteTokenFetcher: func(c config.Component) func() (string, error) {
 				return func() (string, error) { return security.FetchAuthToken(c) }
 			},
-			RemoteFilter: taggerTypes.NewMatchAllFilter(),
+			RemoteFilter:     taggerTypes.NewMatchAllFilter(),
+			ContinueOnErrors: true,
```

Comment:
Would you agree that it be better to override `RemoteTokenFetcher` instead? That would support the same use case without adding a new parameter.

Reply:
I finally took the same approach as a recent PR for other sub-agents: create the token if not present